Make notifications time sensitive

Kloenk opened the pull request:
## Description
Notifications are delivered as time sensitive as it affects the users health.
Fixes: #3559


## Link to Jira
[EXPOSUREAPP-9690](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-9690)

marcussc commented:
@Kloenk thanks for you contribution, one small linter issue needs to be fixed before we can merge your change: https://89576-268042230-gh.circle-artifacts.com/0/src/xcode/swiftlint.html

Kloenk commented:
> @Kloenk thanks for you contribution, one small linter issue needs to be fixed before we can merge your change: https://89576-268042230-gh.circle-artifacts.com/0/src/xcode/swiftlint.html

Thanks, missed that, SwiftLint does not want to run on my machine. Rebased and removed the lint issue.
